packages/dev-frontend/src/components/Bonds/context/api.ts

```diff
@@ -210,6 +252,27 @@ const getProtocolInfo = async (
     total: decimalify(pending.add(reserve).add(permanent))
   };
 
+  if (cachedYearnApys.lusd3Crv === undefined || cachedYearnApys.stabilityPool === undefined) {
+    await cacheYearnVaultApys();
+  }
+
+  let yieldAmplification: Maybe<Decimal> = undefined;
+  let bLusdApr: Maybe<Decimal> = undefined;
+
+  if (
+    cachedYearnApys.lusd3Crv !== undefined &&
+    cachedYearnApys.stabilityPool !== undefined &&
+    bLusdSupply.gt(0)
+  ) {
+    const pendingAndReserveYield = cachedYearnApys.stabilityPool.mul(
+      treasury.pending.add(treasury.reserve)
+    );
+    const permanentYield = cachedYearnApys.lusd3Crv.mul(treasury.permanent);
+    const overallApr = pendingAndReserveYield.add(permanentYield).div(bLusdSupply);
```

Comment:
Shouldn't this be divided by `treasury.reserve`?

Reply:
How come?

Its basically adding the yield sources and working out how much yield 1 bLUSD is receiving in comparison to staking 1 LUSD in the Stability Pool yourself.

Reply:
I guess it depends on how we want to define yield amplification.

Consider this: let's say the system runs long enough for the backing ratio to increase to 3x. Let's also say bonding is no longer attractive, as such there's no pending bucket. The size of the permanent bucket is 10% that of the reserve. For simplicity, let's say both buckets earn 10% APR. What's the yield amplification in this case?

If we divide by bLUSD supply: 3.3X
If we divide by the reserve: 1.1X

Reply:
Yeah that's a good point. I guess the reserve controls the min market price too (redemption price), not the bLUSD supply.

I'll switch it to the reserve now, thanks!

Reply:
Goerli with the adjusted formula:

![image](https://user-images.githubusercontent.com/5167260/192731583-6e4a24fa-2cdb-486e-bfa5-41c6cfef727d.png)